edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -512,6 +512,34 @@ Generation of groups is an organizer responsibility, but as a Delegate it is you
 
 For non-first rounds, it is a good practice to put all of the similarly skilled competitors in one group to ensure that they receive the same scrambles. Use the previous round’s results to determine which groups to place competitors.
 
+## Competition Cancellation and Rescheduling
+
+Once a competition is announced, it is the expectation and responsibility of the Delegates to commit to seeing the competition through on the original date announced. Competitors and their families will have made arrangements and financial commitments to attend on that day, and it is in everyone’s best interest to honor those commitments.
+
+A cancellation should only happen when there is no safe or feasible alternative. This is only an option in extenuating circumstances such as:
+
+- Extreme weather conditions/natural disasters
+- Political unrest or conflict
+- Venue facility failures (electrical, plumbing, structural), where another venue is not possible
+- General safety concerns with competitors traveling to or attending the competition
+- Force majeure
+
+Competitions may not be canceled for the following reasons:
+
+- Low registration numbers
```

Comment:
Technically, a competition with one person registered is not allowed, haha. May be too nitpicky to even mention though, so probably just leave it.

Reply:
I think if that ever occurs there will need to be a larger discussion anyway, with WCAT/Board/Senior of that region as to why a comp was held in a place where nobody signed up.